controllers/app_controller.js

```diff
@@ -1,9 +1,18 @@
-const { fetchCategories } = require('../models/app_model')
+const { 
+    fetchCategories,
+    fetchReviews
+} = require('../models/app_model')
 
 const getCategories = (req, res, next) => {
     fetchCategories().then((categories) => {
         res.status(200).send(categories)
     }).catch(next)
 }
 
-module.exports = { getCategories }
\ No newline at end of file
+const getReviews = (req, res, next) => {
+    fetchReviews().then((reviews) => {
+        res.status(200).send(reviews)
+    })
```

Comment:
Really we should be sending the reviews back on a key of 'review' or something to that effect.